src/main/java/androidx/core/view/View.kt

```diff
@@ -286,3 +290,26 @@ inline fun <reified T : ViewGroup.LayoutParams> View.updateLayoutParams(block: T
     block(params)
     layoutParams = params
 }
+
+/**
+ * Executes the block only when the [duration] longer than the duration of two click timestamp
+ * default debounce time is 1 second
+ * @param duration [Long]
+ * @param timeUnit [TimeUnit]
+ */
+fun View.setDebounceClickListener(
```

Comment:
android-ktx's goal is not to add new features. Such a feature should be submitted to AOSP.